src/sentry/static/sentry/app/components/hovercard.jsx

```diff
@@ -41,6 +41,14 @@ class Hovercard extends React.Component {
      * Position tooltip should take relative to the child element
      */
     position: PropTypes.oneOf(VALID_DIRECTIONS),
+    /**
+     * If set, is used INSTEAD OF the hover action to determine whether the hovercard is shown
+     */
+    show: PropTypes.bool,
```

Comment:
I'm thinking a better name *might* be `isVisible`, but I'm fine with this.. something about `show` feels a little ambiguous to me, but I can't quite put my finger on it.

Reply:
i'd thought about this -- my thinking was `isVisible` seems to denote more the *initial state* of the hovercard, whereas `show` seemed more suitable for the *permanent* state